cli/tests/integrations/test_marathon.py

```diff
@@ -203,6 +203,17 @@ def test_remove_app():
     _list_apps()
 
 
+def test_add_multiapp():
+    with _zero_instance_multiapp():
+        _list_apps(['zero-instance-app1', 'zero-instance-app2'])
+
```

Comment:
Some tests I was thinking of: a multiple apps test when 1 app already exists but the other doesn't, when both/all of the apps already exist, when none of the apps exist, but one has an error.